CHANGES.md

```diff
@@ -1,5 +1,28 @@
 # Changes
 
+## stable-2.13.2
+
+This stable release fixes an incompatibility issue with the AWS CNI addon in EKS
+that was forbidding pods to acquire networking after scaling up nodes (thanks
+@frimik!). It also includes security updates for dependencies.
+
+* CNI
+  * Fixed incompatibility issue with AWS CNI addon in EKS, that was forbidding
+    pods to acquire networking after scaling up nodes. (thanks @frimik!)
+
+* CLI
+  * Fixed `linkerd uninstall` issue for HttpRoute
+
+* Proxy
+  * Updated the dependency on h2 to fix a potential crash in the HTTP/2
+    implementation.
+  * Changed the proxy's default log level to silence warnings from
+    `trust_dns_proto` that are generally spurious
+
+* Extensions
+  * Bumped Prometheus image to v2.43.0
+  * Fixed Jaeger chart installation failure
```

Comment:
Should we specify under which conditions installation could fail?

Reply:
It would always fail. I've rephrased to make it clearer.